src/pow_fpga_accel.c

```diff
@@ -41,7 +41,11 @@ static bool PoWFPGAAccel(void *pow_ctx)
     Trits_t *object_trits = trits_from_trytes(object_trytes);
     if (!object_trits)
         return false;
-
+    
+    lseek(ctx->in_fd,0,0);
+    lseek(ctx->ctrl_fd,0,0);
+    lseek(ctx->out_fd,0,0);
```

Comment:
So what the reason results in this issue ?

Reply:
This is because "The adjustment of the file offset and the write operation are performed as an atomic step" [0]. Therefore, the file offset is not correct when reentering  PoWFPGAAccel function.

[0] http://man7.org/linux/man-pages/man2/write.2.html

Reply:
I don't think "write operation are performed as atomic step" is related to this issue, while in your implementation only one thread could enter the PoWFPGAAccel function.

Reply:
I understand the problem is that the file offset is modified. Hence, it need to be initialized before doing FPGA Pow. Please explain the problem and solution in this Pull Request Description.